### PR TITLE
Dagger CLI version 0.9.6 / x64 & arm64 & arm

### PR DESCRIPTION
**Pull request by nunix**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/135188)

### CODE REVIEW

**Comment by marcosnils** [comment]
that's awesome @nunix. How do we go about keeping this updated with our CI pipeline? Does Goreleaser support this @caarlos0? Since we're using it at @dagger_io

**Comment by caarlos0** [comment]
it does yes!

**Comment by marcosnils** [comment]
cc @gerhard @matipan

File: manifests/d/Dagger/Cli/0.9.6/Dagger.Cli.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Dagger.Cli
```
Comment (stephengillie) [inline manifests/d/Dagger/Cli/0.9.6/Dagger.Cli.installer.yaml:4]:
```suggestion
PackageIdentifier: Dagger.DaggerCli
```
The name "cli" is somewhat generic - please consider a more specific name. 
- Please remember to update the file path and file name to match this change.

File: manifests/d/Dagger/Cli/0.9.6/Dagger.Cli.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Dagger.Cli
```
Comment (stephengillie) [inline manifests/d/Dagger/Cli/0.9.6/Dagger.Cli.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: Dagger.DaggerCli
```
- Please remember to update the file path and file name to match this change.

File: manifests/d/Dagger/Cli/0.9.6/Dagger.Cli.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.5.0.schema.json
+
+PackageIdentifier: Dagger.Cli
```
Comment (stephengillie) [inline manifests/d/Dagger/Cli/0.9.6/Dagger.Cli.yaml:4]:
```suggestion
PackageIdentifier: Dagger.DaggerCli
```
- Please remember to update the file path and file name to match this change.

**Comment by stephengillie** [comment]
Hi @nunix,

This package will show up as `Cli` in searches such as `winget search` and `Find-WinGetPackage`. It might be able to be found by searching `dagger` but having a more specific name can help your users to find your package, and help users looking for other CLI applications to find theirs. 

This is just a suggestion. Let me know what you think.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**